recipe/activate.sh

```diff
@@ -23,6 +23,17 @@ else
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${CONDA_PREFIX}/${targetsDir}/lib/stubs"
 fi
 
+# For conda-build we add the host requirements prefix to the include and link
+# paths for nvcc because it is separate from the build prefix where nvcc is
+# installed.
+if [ "${CONDA_BUILD:-0}" = "1" ]; then
```

Comment:
So in the case where conda-build is not used (like a dev environment), this is able to rely on the existing `nvcc.profile` logic?

Reply:
Correct, these are only needed when there are headers and libs located outside the prefix where nvcc is installed.